mdoc/src/main/scala/mdoc/internal/cli/Settings.scala

```diff
@@ -198,13 +215,12 @@ case class Settings(
       ConfError.fileDoesNotExist(in.toNIO).notOk
     }
   }
-  def resolveIn(relpath: RelativePath): AbsolutePath = {
-    in.resolve(relpath)
-  }
 
-  def resolveOut(relpath: RelativePath): AbsolutePath = {
-    out.resolve(relpath)
+  private def assumedRegularFile(absPath: AbsolutePath): Boolean = {
```

Comment:
So I realized pretty quickly that it was going to be a bit hard to "validate" the `--out` now that it can be a file. Since that "file" won't exist, we can't really check if it's actually a file or now. This is sort of a best effort to ensure that if we assume what was passed into `--out` is an acceptable file (md or html by default), then we check to make sure that the `--in` was also a file.